backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/attribute/impl/HUUniqueAttributesService.java

```diff
@@ -108,6 +109,19 @@ public void validateHU(@NonNull final HuId huId)
 				.forEach(this::validateHUUniqueAttribute);
 	}
 
+	public void validateAttribute(@NonNull final I_M_Attribute attribute)
+	{
+		Services.get(IQueryBL.class).createQueryBuilder(I_M_HU_Attribute.class)
+				.addOnlyActiveRecordsFilter()
+				.addEqualsFilter(I_M_HU_Attribute.COLUMNNAME_M_Attribute_ID, attribute.getM_Attribute_ID())
+				.addEqualsFilter(I_M_HU_Attribute.COLUMNNAME_Value, attribute.getValue())
```

Comment:
> .addEqualsFilter(I_M_HU_Attribute.COLUMNNAME_Value, attribute.getValue())

this looks weird... @metas-rc  wdyt?

Reply:
@riadmen (cc @teosarca ) no, this is not right. The M_Attribute.Value is a string identifier of the attribute , e.g. "SerialNo" while the M_HU_Attribute.Value is the value that attribute receives for a certain HU, e.g. "abc12345"